usaspending_api/awards/migrations/0066_remove_brokersubaward_imported.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-09-30 12:18
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('awards', '0065_dropping_unused_award_columns'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            ["drop index if exists idx_broker_subaward_imported_true"],
+            ["""
+                create index if not exists idx_broker_subaward_imported_true on 
+                broker_subaward (imported) where imported is true"""],
+        ),
+        migrations.RemoveField(
+            model_name='brokersubaward',
+            name='imported',
+        ),
```

Comment:
Should there be a revert step in the operations?

Reply:
I'm pretty sure the built-in migration functions revert automatically.  We don't write code to reverse field additions.